contracts/payloads/IGP27/PayloadIGP27.sol

```diff
@@ -0,0 +1,453 @@
+    address public constant wstETH_ADDRESS =
+        0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+    address public constant weETH_ADDRESS =
+        0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee;
+
+    address public constant sUSDe_ADDRESS =
+        0x9D39A5DE30e57443BfF2A8307A4256c8797A3497;
+    address public constant USDC_ADDRESS =
+        0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address public constant USDT_ADDRESS =
+        0xdAC17F958D2ee523a2206206994597C13D831ec7;
+
+    constructor() {
+        ADDRESS_THIS = address(this);
+    }
+
+    function propose(string memory description) external {
+        require(
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Update rewards for fUSDC & fUSDT.
+        action1();
+
+        // Action 2: Update USDC, USDT & wstETH market rate curve.
+        action2();
+
+        // Action 3: Set max utilization to 85% for wstETH & Update Reserve fee to 10%.
+        action3();
+    }
+
+    function verifyProposal() external view {}
+
+    /***********************************|
+    |     Proposal Payload Actions      |
+    |__________________________________*/
+
+    /// @notice Action 1: Update rewards for fUSDC & fUSDT.
+    function action1() internal {
+        IFTokenAdmin(F_USDC).updateRewards(
+            0xf6109Ad48f5Cf9e28938774aD4069C90f81D648E
+        );
+
+        IFTokenAdmin(F_USDT).updateRewards(
+            0xf6109Ad48f5Cf9e28938774aD4069C90f81D648E
+        );
+    }
+
+    /// @notice Action 2: Update USDC, USDT & wstETH market rate curve.
+    function action2() internal {
+        AdminModuleStructs.RateDataV2Params[]
+            memory params_ = new AdminModuleStructs.RateDataV2Params[](3);
+
+        params_[0] = AdminModuleStructs.RateDataV2Params({
```

Comment:
rates here for USDC and USDT need to be updated according to https://gov.instadapp.io/t/update-usdc-usdt-and-wsteth-market-rates-on-fluid-jun-24-2024/780

currently we are setting "Current rates" instead of proposed rates (0, 10, 15, 25)